custom_components/svt_play/__init__.py

```diff
@@ -12,6 +12,7 @@
 CONF_PROGRAM_NAME = 'program_name'
 CONF_CHANNEL = 'channel'
 CONF_CATEGORY = 'category'
+CONF_SKIP = 'skip'
```

Comment:
I think exclude will be the best naming here since we could then add `include` later with a similar naming scheme.